stack.yaml

```diff
@@ -55,7 +55,7 @@ nix:
 allow-newer: true
 
 extra-deps:
-- swagger2-2.4
+# - swagger2-2.4
```

Comment:
can this be removed? Or why is it commented?

Reply:
Hmm, I forgot a few comments, I will clean them up, thanks!